package.json

```diff
@@ -13,7 +13,6 @@
   },
   "homepage": "https://redux-toolkit.js.org",
   "directories": {
-    "doc": "docs",
```

Comment:
This was blocking the install of dependencies for docs - so if you cloned this repo, ran `yarn` and then tried to start the docs, it'd error with a missing formik dependency.